lib/mongo/usersToObject.js

```diff
@@ -0,0 +1,11 @@
+function userAllDataToObject(userObject) {
+    let user = {}
+    user.courseName = userObject.name
+    user.email = userObject.email
+    user.id = userObject.id
+    return user
```

Comment:
Just checking, but do you need the courseIds and chatIds? Might be useful so you don't have to go searching for it again.